bazel/shellutils.bzl

```diff
@@ -0,0 +1,57 @@
+# Bazel rules for helping with shell scripts.
+
+def sh_escape(x):
```

Comment:
use skylib, shell, _quote? (we use skylib in other places already), remove this function?

Reply:
Done.